Makefile

```diff
@@ -1,13 +1,22 @@
 GO_DOCKER_RUN = docker run --rm -v $(shell pwd):/go/src/github.com/nginxinc/nginx-asg-sync -v $(shell pwd)/build_output:/build_output -w /go/src/github.com/nginxinc/nginx-asg-sync/cmd/sync
 GOLANG_CONTAINER = golang:1.10
+BUILD_IN_CONTAINER ?= 1
 
 all: amazon centos7 ubuntu-trusty ubuntu-xenial
 
 test:
+ifeq ($(BUILD_IN_CONTAINER),1)
 	$(GO_DOCKER_RUN) $(GOLANG_CONTAINER) go test
+else
+	go test ./...
+endif
 
 compile: test
+ifeq ($(BUILD_IN_CONTAINER),1)
 	$(GO_DOCKER_RUN) $(GOLANG_CONTAINER) go build -o /build_output/nginx-asg-sync
+else
+	go build -v ./...
```

Comment:
make sure that `go build -v ./...` produced the output in the /build_output with the name `nginx-asg-sync`

Reply:
otherwise, `make BUILD_IN_CONTAINER=0` will fail

Reply:
yes, you are right, since I was not building the packages for this pipeline I did not realise about that.
Changed now